packages/loot-core/src/mocks/index.js

```diff
@@ -55,9 +55,8 @@ function _generateTransaction(data) {
   return {
     id: id,
     amount: data.amount || Math.floor(Math.random() * 10000 - 7000),
-    payee: data.payee || (Math.random() < 0.9 ? 'payed-to' : 'guy'),
-    notes:
-      Math.random() < 0.1 ? 'A really long note that should overflow' : 'Notes',
+    payee: data.payee || 'payed-to',
+    notes: 'Notes',
```

Comment:
Is there a way to test that these cases are still handled properly? (overflow)

Reply:
No. And we didn't have one before either.

Text overflow is a visual aspect of the product that should be tested with visual regression tests (screenshots). But we have none of those.

Reply:
Ok, sounds good!